File: frontend/src/app/models/user.model.ts
export class User{
    user_type: string;
    first_name: string;
    last_name: string;
    email: string;
    password_hash: string;
    usr_pho: string;
    constructor(){
        this.email = '';
        this.user_type = '';
        this.first_name = '';
        this.last_name = '';
        this.password_hash = '';
        this.usr_pho = '';
    }
}

File: backend/APIs/Models/UserModel.cs
using System.ComponentModel.DataAnnotations;
namespace APIs.Models {
    public class UserModel {

        [Key]
        public Int64 user_id { get; set; }
        public string user_type { get; set; }
        public string first_name { get; set; }
        public string last_name { get; set; }
        public string email { get; set; }
        public string password_hash { get; set; }
        public string usr_pho {  get; set; }
    }
}
